inc/button_interface.hpp

```diff
@@ -52,6 +52,7 @@ class ButtonIface
             buttonIface->handleEvent(es, fd, revents);
             return 0;
         }
+        return 0;
```

Comment:
there's a gerrit commit at https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-buttons/+/51183/3/inc/button_interface.hpp#55 that also removes the return at line 53